projects.js

```diff
@@ -0,0 +1,165 @@
+    languages: ['html', 'css', 'javascript'],
+    imageUrl: 'images/work2.svg',
+    id: '2',
+  },
+  {
+    title: 'Multi-Post Stories',
+    company: 'CANOPY',
+    position: 'Back End Dev',
+    date: '2015',
+    desc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    languages: ['html', 'css', 'javascript'],
+    imageUrl: './images/work3.svg',
+    id: '3',
+  },
+  {
+    title: 'Multi-Post Stories',
+    company: 'Tonic',
+    position: 'Back End Dev',
+    date: '2015',
+    desc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
```

Comment:
- [ ] I have noticed that the container of this text has a fixed height, thus the text is goig out of container border, please fix this issue: 
![image](https://user-images.githubusercontent.com/77038610/151207450-ed1602f4-9b26-42cf-a0e3-3f59b44bbb68.png)